static/style.css

```diff
@@ -100,6 +110,13 @@ input[type="text"]:focus {
 }
  section.footer:hover {
 	opacity: 1;
+
```

Comment:
Blank lines inside braces are in none of all recommendations. However most recommends adding an empty line between declarations.

Reply:
I don't think we have any guidelines on CSS formatting, so I just kept the format that was used that time. It has changed since, so I need to re-adjust. Thanks for noticing :+1: